httpcore/_async/http11.py

```diff
@@ -191,8 +195,11 @@ async def _receive_event(self, timeout: Optional[float] = None) -> H11Event:
                     raise RemoteProtocolError(msg)
 
                 self._h11_state.receive_data(data)
+            elif event is h11.PAUSED:
+                # TODO: Implement handling for paused
```

Comment:
h11.PAUSED is handled in https://github.com/encode/httpcore/pull/526/files#diff-f1329e0e83048035c0a643ab5f3c666ae6565cba6393111fc0dc1afc4f2dbaa4R168